Registry/AssetProcessorPlatformConfig.setreg

```diff
@@ -76,7 +76,8 @@
                     "animsettings": "i_caf",
                     "Animations/SkeletonList.xml": "i_caf",
                     "cbc": "abc",
-                    "fbx.assetinfo": "fbx"
+                    "fbx.assetinfo": "fbx",
+                    "stl.assetinfo": "stl"
```

Comment:
This is an OK quick fix for Stabilization, but I don't think we should bring this to dev. We should use the same list of extensions that drive what the scene system loads, so the next scene file added doesn't have this bug.

Reply:
The PR author and I talked over Slack about this. This doesn't hurt anything to get in as-is, and fixes a bug. The author's going to look into a data driven fix next, that uses the scene file list from o3de\Registry\sceneassetimporter.setreg.